services/app-web/src/testHelpers/apolloHelpers.tsx

```diff
@@ -836,25 +852,94 @@ const mockSubmittedHealthPlanPackageWithRevision = (): HealthPlanPackage => {
     }
 }
 
-type updateHealthPlanFormDataMockSuccessProps = {
+type updateHealthPlanFormDataMockProps = {
     pkg?: HealthPlanPackage
     updatedFormData: string
     id: string
+    statusCode?: 200 | 403 | 404 | 500
 }
 
-const updateHealthPlanFormDataMockSuccess = ({
+const updateHealthPlanFormDataMock = ({
```

Comment:
I’ve been moving away from these switch based helpers to individual helpers. “updateHPFDMockSuccesss”, “updateHPFDMockAuthFailure”, “updateHPFDMockNotFound”

it’s slightly more repeated code at the top of each function but I think is a bit more clear. I might need to go through and consolidate that.